src/pages/Home/home.ts

```diff
@@ -391,7 +310,9 @@ export class HomeViewModel {
     }
 
 
-
+    showModal2(event) {
```

Comment:
remove the showModal and rename this method? It's really confusing to have a showModal variable and a showModal2 method